host/dynamicconfig.go

```diff
@@ -78,6 +78,10 @@ func (d *dynamicClient) GetDurationValue(
 	return d.client.GetDurationValue(name, filters, defaultValue)
 }
 
+func (d *dynamicClient) UpdateValue(name dynamicconfig.Key, value interface{}) error {
+	return nil
```

Comment:
return not implemented error?

Also do we have a plan to add an integration test to make sure this command works?

Reply:
Updated. 
I do have plan to add unit tests (maybe also integration tests) on all Admin APIs